src/main/java/uk/gov/hmcts/reform/pip/data/management/controllers/LocationController.java

```diff
@@ -106,11 +108,12 @@ public ResponseEntity<Collection<Location>> uploadLocations(@RequestPart Multipa
         @ApiResponse(responseCode = AUTH_ERROR_CODE, description = "User has not been authorized"),
         @ApiResponse(responseCode = NOT_FOUND_CODE, description = "No Location found with the id {locationId}")
     })
-    @DeleteMapping("/{locationId}")
+    @DeleteMapping("/{locationId}/{requesterName}")
```

Comment:
The path of an endpoint should ideally be just showing information required for its usage. In this case it is to delete a location by the location ID. May be requesterName should not be part of the path. Can consider adding that to the header?

Reply:
Yes, you are right. I have updated the code.